defaults/ekscluster-kfd-v1alpha2.yaml

```diff
@@ -256,6 +256,7 @@ data:
           cron:
             manifests: ""
             full: ""
+          ttl: ""
```

Comment:
IMO, having this defaults here is confusing because they are not the actual default values. See my other comment in the template file.